news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title: osu!monthly #1
+date: 2022-07-28
+author: Soetch, Blushing, wowcake, 0x84f
+---
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)
+
+# Introducing osu!monthly
+
+It has been a long time since the last osu!weekly was released and luckily, we have something exciting to show you: osu!monthly! The new community news post format.
+
+6 years passed in which the community didn't have a regular post. Today, we're coming in with ***osu!monthly***, a bigger format.
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)
+
+## Word of the editors :
+
+It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post. Today, we're coming in with **__osu!monthly__**, a bigger format.
```

Comment:
```suggestion
```
My suggestion was to shorten this to the sentence above (`6 years passed in which...`) and have everything under `Introducing osu!monthly` (`word of the editors` needs to be removed too in this case)

Reply:
Doing this rn, sorry for not noticing.

Reply:
Actually, I'm keeping the 2 last paragraphs of it, see the last commit.